pkg/reconciler/persistentvolumeclaim/metadata.go

```diff
@@ -150,8 +150,7 @@ func newAnnotationReconciler(cluster *apiv1.Cluster) metadataReconciler {
 			return utils.IsAnnotationSubset(pvc.Annotations,
 				cluster.Annotations,
 				cluster.GetFixedInheritedAnnotations(),
-				configuration.Current) &&
```

Comment:
we were checking for appArmor in the PVC reconciliation, I removed it because it made no sense.

We didn't add appArmor annotations anyway